setup.py

```diff
@@ -65,4 +65,36 @@ def get_extensions():
     return [erfa_ext]
 
 
-setuptools.setup(ext_modules=get_extensions())
+VERSION_TEMPLATE = """
+'''Wrapper, ERFA and SOFA version information.'''
+
+# Set the version numbers a bit indirectly, so that Sphinx can pick up
+# up the docstrings and list the values.
+from . import ufunc
+
+
+erfa_version = ufunc.erfa_version
+'''Version of the C ERFA library that is wrapped.'''
+
+sofa_version = ufunc.sofa_version
+'''Version of the SOFA library the ERFA library is based on.'''
+
+del ufunc
+
+# Note that we need to fall back to the hard-coded version if either
+# setuptools_scm can't be imported or setuptools_scm can't determine the
+# version, so we catch the generic 'Exception'.
+try:
+    from setuptools_scm import get_version
+    version = get_version(root='..', relative_to=__file__)
+    '''Version of the python wrappers.'''
+except Exception:
+    version = '{version}'
+else:
+    del get_version
+""".lstrip()
```

Comment:
If you want you could also move the `VERSION_TEMPLATE` contents to a `erfa/version.py.templ` file, bit also as it is now is OK for me.

Reply:
I just copied this method over from astropy. My own sense is to stay close to that since most of us are used to it...

Reply:
OK, makes sense